### DISCUSSION TITLE
开发时如何在电脑端也能收起侧边栏？

### DISCUSSION THREAD

**Discussion by skeetyu** [vuepress-theme-hope/vuepress-theme-hope#3769] [Q&A] [answered]
响应式布局中提到侧边栏在平板端是处于可折叠状态的，区分设备的方式似乎是按照分辨率的大小？现在我使用theme-hope进行开发，希望能够在电脑端也启用“侧边栏的折叠”，应该怎么设置呢？
<img width="466" alt="截屏2023-12-18 12 54 49" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/62090366/5bdc98f8-4557-45aa-b16c-71e29a169ba9">

**Comment by Mister-Hope** [comment, marked as answer]
设置断点

**Reply by skeetyu** [reply]
感谢回复！
我目前设置了$pc, $tablet等断点，确实实现了一定效果。但是“当前页面的标题”也被收入侧边栏中了。
我希望能在实现“侧边栏折叠”的同时，依旧让“当前页面的标题显示在页面右侧”，不知道这可以做到吗？

**Reply by Mister-Hope** [reply]
自己改样式

**Reply by skeetyu** [reply]
请问是要自己写scss覆盖原有的默认样式吗？方便问问原有样式是在哪个文件下配置的吗？

**Reply by Mister-Hope** [reply]
navbar部分
